src/goodbase/goodbase_cli.py

```diff
@@ -441,25 +438,20 @@ def main(
         output_format=output_format,
     )
 
-    display_variant_name_checks = display_variant_name
-
-    if ctx.get_parameter_source("display_variant_name") == ParameterSource.DEFAULT:
-        for (
-            project_regex,
-            default_display_variant_name_regexes,
-        ) in PROJECT_REGEX_TO_DEFAULT_DISPLAY_VARIANT_NAME_REGEXES.items():
-            if re.match(project_regex, evg_project):
-                LOGGER.debug(
-                    "Found default build variant display name regexes for evg project",
-                    evg_project_regex=project_regex,
-                    default_display_variant_name_regexes=default_display_variant_name_regexes,
-                )
-                display_variant_name_checks = default_display_variant_name_regexes
-                break
```

Comment:
Why are we removing this for mongo defaults?

Reply:
Added this back.